shared/ssa/qlpack.yml

```diff
@@ -1,4 +1,4 @@
-name: codeql/shared-ssa
+name: codeql/ssa
 version: 0.0.1-dev
```

Comment:
This will have to be manually changed to `0.0.1` before releasing and then manually bumped to `0.0.2` (or whatever) after releasing. We can implement automation for this in a similar way to how we automate the core libraries. 

For now, the steps should be:

1. remove the `-dev` suffix
2. commit the change
3. `codeql pack publish`
4. bump the version and add the `-dev` back
5. bump all dependencies in the libraries as well
6. commit the change

This simulates the automation will do.